### ISSUE TITLE
Stall after successfully selecting a build

### ISSUE THREAD

**Issue by rogerluan** [fastlane/fastlane#16648]
### New Release Candidate Issue Checklist

- [x] Updated fastlane to the latest release candidate version (2.150.0.rcX)
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Information

- Breaking version: 2.150.0.rc1
- Last working version: Latest version before June 16th
- State of app: `Prepare for Submission`
- First version of app?: No

### Issue Description

I invoked `deliver` with "wait for processing". After the binary finished processing, it got stuck in the `successfully selecting a build` message. It's been stuck for ~20 minutes now:

<img width="791" alt="image" src="https://user-images.githubusercontent.com/8419048/85335565-97f84f80-b4b3-11ea-89a6-f0eb742615b8.png">

### Spaceship Logs

<details>
  <pre>removed</pre>
</details>

### Environment

<details>
  <pre><details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                           |
| --------------------------- | --------------------------------------------------------------- |
| OS                          | 10.15.5                                                         |
| Ruby                        | 2.6.5                                                           |
| Bundler?                    | true                                                            |
| Git                         | git version 2.24.3 (Apple Git-128)                              |
| Installation Source         | ~/Documents/Projects/tellus-ios/.vendor/ruby/2.6.0/bin/fastlane |
| Host                        | Mac OS X 10.15.5 (19F101)                                       |
| Ruby Lib Dir                | ~/.rbenv/versions/2.6.5/lib                                     |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                                     |
| Is contained                | false                                                           |
| Is homebrew                 | false                                                           |
| Is installed via Fabric.app | false                                                           |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                     |
| Xcode Version               | 11.5                                                            |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile.swift`</summary>

```ruby
I removed it
```
</details>

**No Appfile found**

### fastlane gems

| Gem      | Version     | Update-Status |
| -------- | ----------- | ------------- |
| fastlane | 2.150.0.rc1 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| bundler                   | 2.1.4        |
| CFPropertyList            | 3.0.2        |
| ZenTest                   | 4.12.0       |
| RubyInline                | 3.12.5       |
| concurrent-ruby           | 1.1.6        |
| i18n                      | 0.9.5        |
| minitest                  | 5.14.0       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.6        |
| activesupport             | 4.2.11.1     |
| public_suffix             | 4.0.5        |
| addressable               | 2.7.0        |
| httpclient                | 2.8.3        |
| json                      | 2.3.0        |
| algoliasearch             | 1.27.1       |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.332.0      |
| aws-sigv4                 | 1.2.0        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.100.0      |
| aws-sdk-kms               | 1.34.1       |
| aws-sdk-s3                | 1.69.1       |
| babosa                    | 1.0.3        |
| claide                    | 1.0.3        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| netrc                     | 0.11.0       |
| ffi                       | 1.12.2       |
| ethon                     | 0.12.0       |
| typhoeus                  | 1.3.1        |
| cocoapods-core            | 1.9.1        |
| cocoapods-deintegrate     | 1.0.4        |
| cocoapods-downloader      | 1.3.0        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.1.0        |
| cocoapods-trunk           | 1.4.1        |
| cocoapods-try             | 1.1.0        |
| colored2                  | 3.1.2        |
| escape                    | 0.0.4        |
| fourflusher               | 2.3.1        |
| gh_inspector              | 1.1.3        |
| molinillo                 | 0.6.6        |
| ruby-macho                | 1.4.0        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.16.0       |
| cocoapods                 | 1.9.1        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| cocoapods-binary          | 0.4.4        |
| dotenv                    | 2.7.5        |
| osx_keychain              | 1.0.2        |
| cocoapods-keys            | 2.1.0        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| digest-crc                | 0.5.1        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| emoji_regex               | 1.0.1        |
| excon                     | 0.75.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.1.7        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.2       |
| multi_json                | 1.14.1       |
| os                        | 1.1.0        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.0       |
| mini_mime                 | 1.0.2        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.2        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.26.2       |
| mini_magick               | 4.10.1       |
| multi_xml                 | 0.6.0        |
| plist                     | 3.5.0        |
| rubyzip                   | 1.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.8.0        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| xcpretty-travis-formatter | 1.0.0        |
| mustache                  | 1.1.1        |
| open4                     | 1.3.4        |
| redcarpet                 | 3.5.0        |
| sassc                     | 2.2.1        |
| sqlite3                   | 1.4.2        |
| liferaft                  | 0.0.6        |
| xcinvoke                  | 0.3.0        |
| jazzy                     | 0.13.1       |
| psych                     | 3.0.3        |
</details>

*generated on:* **2020-06-22**
</details></pre>
</details>

### Mentioning

cc: @joshdholtz

**Comment by rogerluan** [comment]
Now it's not stalling anymore, instead it errors:

```
Spaceship::UnexpectedResponse: [!] The request cannot be fulfilled because of the state of another resource. - Submit for review errors found.
The provided entity is missing a required attribute - You must provide a value for the attribute 'usesIdfa' with this request
```

Prior to the App Store Connect changes this used to work (fastlane Swift):

```
/// Extra information for the submission (e.g. compliance specifications, IDFA settings).
    var submissionInformation: Any? {
        return [
            "add_id_info_limits_tracking" : true,
            "add_id_info_serves_ads" : false,
            "add_id_info_tracks_action" : true,
            "add_id_info_tracks_install" : true,
            "add_id_info_uses_idfa" : true,
            "content_rights_has_rights" : true,
            "content_rights_contains_third_party_content" : true,
            "export_compliance_platform" : "ios",
            "export_compliance_compliance_required" : false,
            "export_compliance_encryption_updated" : false,
            "export_compliance_app_type" : nil,
            "export_compliance_uses_encryption" : false,
            "export_compliance_is_exempt" : false,
            "export_compliance_contains_third_party_cryptography" : false,
            "export_compliance_contains_proprietary_cryptography" : false,
            "export_compliance_available_on_french_store" : false
        ]
    }
```

I didn't update fastlane since my last report above (there hasn't been any release since then)

Hope this helps! Let me know if you need more info.

**Comment by joshdholtz** [comment]
@rogerluan Hey 👋  Can you try updating to `2.150.0.rc2` and see if things are fixed for you now? ❤️

**Comment by joshdholtz** [comment]
Closing this since this should be fixed now! Latest version is `2.150.1`

**Comment by rogerluan** [comment]
Cheers!
